var/spack/repos/builtin/packages/r-annotationforge/package.py

```diff
@@ -7,19 +7,24 @@
 
 
 class RAnnotationforge(RPackage):
-    """Provides code for generating Annotation packages and
-    their databases. Packages produced are intended to be used
-    with AnnotationDbi."""
+    """Tools for building SQLite-based annotation data packages.
 
-    homepage = "https://www.bioconductor.org/packages/AnnotationForge/"
+       Provides code for generating Annotation packages and their databases.
+       Packages produced are intended to be used with AnnotationDbi."""
+
+    homepage = "https://bioconductor.org/packages/AnnotationForge"
     git      = "https://git.bioconductor.org/packages/AnnotationForge.git"
 
+    version('1.26.0', commit='5d181f32df1fff6446af64a2538a7d25c23fe46e')
+    version('1.24.0', commit='3e1fe863573e5b0f69f35a9ad6aebce11ef83d0d')
+    version('1.22.2', commit='8eafb1690c1c02f6291ccbb38ac633d54b8217f8')
+    version('1.20.0', commit='7b440f1570cb90acce8fe2fa8d3b5ac34f638882')
     version('1.18.2', commit='44ca3d4ef9e9825c14725ffdbbaa57ea059532e1')
 
-    depends_on('r@3.4.0:3.4.9', when='@1.18.2')
-    depends_on('r-biocgenerics', type=('build', 'run'))
-    depends_on('r-biobase', type=('build', 'run'))
-    depends_on('r-annotationdbi', type=('build', 'run'))
+    depends_on('r@2.7.0:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('r@3.5.0:', type=('build', 'run'))
```

Reply:
This is very strange. The Bioconductor website shows a minimum R version of 3.5.0 while the DESCRIPTION in the repository has a minimum version of 2.7.0.

Reply:
Still not sure what we want to do here, I'll let you decide.

Reply:
Let's trust the repository, that's what will actually crash if you try to install with the wrong version of R